build.gradle

```diff
@@ -22,6 +19,10 @@ dependencies {
 
     modImplementation "net.fabricmc.fabric-api:fabric-resource-loader-v0:0.6.1+aeb40ebe90"
     include "net.fabricmc.fabric-api:fabric-resource-loader-v0:0.6.1+aeb40ebe90"
+
+    modApi("me.shedaniel.cloth:cloth-config-fabric:8.3.103") {
+        exclude(group: "net.fabricmc.fabric-api")
+    }
```

Comment:
Is it possible to not force the user to download Cloth Config, and only require that if the user wants to use Mod Menu?

Reply:
It's not for Mod Menu, it's just for a config API. I might use one I can include with shadow instead.

Reply:
It's now shadowed!